src/index.html

```diff
@@ -7,12 +7,12 @@
 </head>
 
 <body>
-  <h1>Leaderboard</h1>
+  <h1> &#127942 Leaderboard &#127942</h1>
   <main class="contenedor">
     <div class="recentScores">
       <div class="refresh">
         <h3>Recent scores</h3>
-        <button class="btn refresh-button">Refresh</button>
+        <button class="btn refresh-button">&#128472</button>
```

Comment:
- [OPTIONAL] I would recommend you to add `Refresh` text inside that button for better user experience 
![your-board](https://user-images.githubusercontent.com/26597432/217946127-78b6285c-e1a6-4a71-93e2-bb0e88eb4505.png)

Reply:
off course, I will change

Reply:
The LICENCE was created, thank you
![image](https://user-images.githubusercontent.com/114632980/217955832-ca14a4c5-fa55-4271-a6cf-955b1ee41db9.png)